### PR TITLE
Adding ruby 3.2.2 to ruby.amazon.properties

### PR DESCRIPTION
**Pull request by zenspider**
Not the latest 3.2 version, but what is listed at:

https://docs.aws.amazon.com/linux/al2023/release-notes/all-packages-AL2023.5.html

### CODE REVIEW

**Comment by zenspider** [comment]
I'm operating by pattern matching... not sure if this is right or not. Feedback welcome.

**Comment by zenspider** [comment]
Previous ruby addition was #2816

**Comment by zenspider** [comment]
@mattgodbolt added https://github.com/compiler-explorer/infra/pull/1352

File: etc/config/ruby.amazon.properties [hunk 1]
```diff
@@ -1,12 +1,14 @@
 compilers=&ruby
-defaultCompiler=ruby302
+defaultCompiler=ruby322
```
Comment (mattgodbolt) [inline etc/config/ruby.amazon.properties:2]: 330 is avilable too should you wish to do an update to this PR later

**Review by mattgodbolt — APPROVED** [review]
Will merge and deploy but 3.3.0 is easy to add now it's installed too. I'm not well placed to know if 3.3.0 is right to be the default etc

**Comment by mattgodbolt** [comment]
3.2.2 is now live

**Pull request merged**